### ISSUE TITLE
CultureOverview shows UBs from other Civs

### ISSUE THREAD

**Issue by Infixo** [LoneGazebo/Community-Patch-DLL#3107]
_1. Mod version (i.e Date - 4/23):_
2/15

_2. Mod list (if using Vox Populi only, leave blank):_
VP with (6a) EUI

_3. Error description:_ 
CultureOverview shows on the top line buildings that are UBs from other Civs. 
Example attached. For Carthage, there are: Burial Tomb (Egypt), Acropolis (Greek), Ceilidh Hall (The Celts), Dojo (Japan)

_4. Steps to reproduce (optional):_

---------------------------
Supporting information:
Please note that you can attach .zip files by dragging-and-dropping them. If possible, zip up all supporting data and post that way.

1. Log files (always attach your Logs folder, located at My Documents/My Games/Sid Meier's Civilization 5. Make sure you have enabled logging before experiencing an error! Go here to find out how: http://forums.civfanatics.com/showthread.php?t=487482):

2. Save game (always attach a save that was made a turn before the error; located at My Documents/My Games/Sid Meier's Civilization 5/ModdedSaves):

3. CvMiniDump.dmp file (attach if experiencing a game crash. Located at Program Files/Steam/steamapps/common/Sid Meier's Civilization V):

4. Screenshots (optional):
![ui culture overview](https://cloud.githubusercontent.com/assets/25866684/23286313/357c6f38-fa37-11e6-845e-0d5e19f47e46.jpg)

**Comment by LoneGazebo** [comment]
Somehow Kyte's fixed code slipped out...

**Comment by Infixo** [comment]
Wow, you are fast. For practice I created and tested a solution, actually a simple one.
In function AssignBuildingToColumn() I've added the below code at the very begining and it works beautifully:

`		if building.CivilizationType ~= nil and building.CivilizationType then
			if building.CivilizationType ~= playerCivilizationType then return; end
		end
`
Also tested a scenario when a civ acquires a city with UB that's from another civ. The UB is displayed correctly on the left side of the window.
